.slsa-goreleaser.yml

```diff
@@ -0,0 +1,13 @@
+version: 1
```

Comment:
Does this need to be its own file? I'm worried about this diverging from goreleaser eventually.

Or, if the idea is to eventually only have the SLSA-flavored goreleaser workflow, I wonder if you've considered just adding the SLSA provenance functionality to goreleaser proper, and not having your own fork?

Reply:
> Does this need to be its own file? I'm worried about this diverging from goreleaser eventually.

right now it needs to be its own file, yes.

> Or, if the idea is to eventually only have the SLSA-flavored goreleaser workflow, 

correct. In the next release we will support multiple builds.

> I wonder if you've considered just adding the SLSA provenance functionality to goreleaser proper, and not having your own fork?

We have (see https://github.com/goreleaser/goreleaser/issues/2737) and we emailed them, but no concrete plan yet :/

Reply:
It sounds like there's still some discussion happening on https://github.com/goreleaser/goreleaser/issues/2737, and I'd kind of like that to land somewhere before proceeding to fork the release process here. If you need any help getting the SLSA provenance generation landed in goreleaser, let me know.

I'd rather not have a forked release process per-target-arch, and long term, I'd prefer to rely on the standard upstream goreleaser if at all possible.

Reply:
> It sounds like there's still some discussion happening on [goreleaser/goreleaser#2737](https://github.com/goreleaser/goreleaser/issues/2737), and I'd kind of like that to land somewhere before proceeding to fork the release process here. If you need any help getting the SLSA provenance generation landed in goreleaser, let me know.

Would love your help :-)

> I'd rather not have a forked release process per-target-arch, and long term, I'd prefer to rely on the standard upstream goreleaser if at all possible.

Any particular reason? We'd like to learn about blockers to adoption to improve. Any feedback welcome.

Reply:
I'm not sure I understand what multi-build support entails, or v2 of what project. In my ideal scenario, goreleaser would be able to be configured to generate and attach attestation documents to releases, which users could use to verify the provenance of each goreleaser-built artifact.

Ideally that's done by just adding `generate-attestations: true` in the goreleaser.yaml somewhere, and no new dependencies on other projects.

Reply:
Sorry for the confusion. What I mean is that in the next release of our SLSA builder, we will support building for multiple arch/os at once, like Goreleaser does. So it would be easy to migrate to our builder instead of Goreleaser with a similar config file.

> Ideally that's done by just adding generate-attestations: true in the goreleaser.yaml somewhere, and no new dependencies on other projects.

Any update on #2737?